app/controllers/edge_controller.rb

```diff
@@ -8,7 +8,38 @@ class EdgeController < RestController
   include ExtractEdgeResources
   include GroupMembershipValidator
 
-  def slosilo_keys
+  def all_authenticators
+    logger.info(LogMessages::Endpoints::EndpointRequested.new("all-authenticators"))
+    allowed_params = %i[account kind limit offset]
+    options = params.permit(*allowed_params).to_h.symbolize_keys
+    begin
+      verify_edge_host(options)
+    rescue ApplicationController::Forbidden
+      raise
+    end
+    account = options[:account]
+
+    allowed_kind = ['authn-jwt']
```

Comment:
Don;t know if its possible in Ruby but I will define it at class level and not method level